index.ts

```diff
@@ -225,6 +297,80 @@ export type BiometricAuthenticateParams = {
   reason?: string;
 };
 
+export type AccelerometerStartParams = {
+  refresh_rate?: number;
+};
+
+export type Accelerometer = {
+  isStarted: boolean;
+  x: number;
+  y: number;
+  z: number;
+  start: (
+    params: AccelerometerStartParams,
+    callback?: (isStarted: boolean) => void
```

Comment:
We use `unknown` instead of `void` type for callbacks that developers may pass. Please change here and in other places.